tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -63,6 +63,8 @@ public static String createXMLNameValuePair( String name, String value )
 
     public static List<String> SUPPORT_SERVER_TYPES = Arrays.asList( new String[]{ "tomcat", "jboss", "glassfish", "jetty"} );
```

Comment:
the list of supported server types should come from the collection of our
portal bundle extension points.  Not hard coded in a string like that. For
example, when we add "wildfly" support, you will hae to edit this
hard-coded string, which isn't maintainable.  So this list should be
calculated dynamically from extension points, not hard-coded.

On Sun, Sep 6, 2015 at 10:06 PM, simon.jiang <notifications@github.com>
wrote:

> In
> tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java
> <https://github.com/gamerson/liferay-ide/pull/1129#discussion_r38831235>:
>
> > @@ -63,6 +63,8 @@ public static String createXMLNameValuePair( String name, String value )
> >
> >      public static List<String> SUPPORT_SERVER_TYPES = Arrays.asList( new String[]{ "tomcat", "jboss", "glassfish", "jetty"} );
>
> I know, do you mean our IDE will dynamically add new server type even if
> customer won't upgrade IDE package?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-ide/pull/1129/files#r38831235>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com